animareflection/ui
# Textarea Component

## Description

##### Task link: https://trello.com/c/QEYe4KGH/126-textarea-component

Added `Textarea` component

## Test Steps

1. Ensure the component's structure, stories, and testing suite are robust.
2. Confirm that all tests are successful.
3. Check that the Textarea displays correctly in both storybook and sample downstream applications.
